packages/react/src/River/River.fixtures.stories.tsx

```diff
@@ -120,6 +120,86 @@ export const AlternativeHeadingSize: StoryFn<typeof River> = () => (
   </Container>
 )
 
+const ExampleTrailingComponentOne = () => (
+  <Stack direction="vertical" padding="none" gap="spacious">
+    <div>
+      <Heading as="h3" size="6">
+        Heading goes here.
+      </Heading>
+      <Text variant="muted" as="p" size="300">
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit. In sapien sit ullamcorper id. Aliquam luctus sed turpis
+        felis nam pulvinar risus elementum.
+      </Text>
+    </div>
+    <div>
+      <Heading as="h3" size="6">
+        Heading goes here.
+      </Heading>
+      <Text variant="muted" as="p" size="300">
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit. In sapien sit ullamcorper id. Aliquam luctus sed turpis
+        felis nam pulvinar risus elementum.
+      </Text>
+    </div>
+  </Stack>
+)
+
+export const CustomTrailingContent: StoryFn<typeof River> = () => (
+  <Container>
+    <River>
+      <River.Visual>
+        <PlaceholderImage />
+      </River.Visual>
+      <River.Content trailingComponent={ExampleTrailingComponentOne}>
```

Comment:
Should we take this opportunity to convert to prop to a child component? I'm fine with either approach, but this seemed like an opportunity to consider.

Reply:
🤔 yeah possibly. 

Reasons for keeping this way:

- we already have the trailingComponent prop, and can therefore avoid a breaking change.
- a dedicated child may give open floodgates to misusing the current API. Right now it behaves more as an escape hatch, via a prop.. making it a dedicated child could lead to something like this:

```jsx
<River>
  <River.TrailingComponent>
    <Heading .../ >
    <Text />
    <Text />
  </River.TrailingComponent>
</River>
```

Granted, you _could_ do this with the prop version, but it _feels_ much more hacky, which is a good thing.